theano/tensor/elemwise.py

```diff
@@ -125,6 +125,7 @@ class DimShuffle(Op):
 
     _f16_ok = True
     check_input = False
+    __props__ = ("input_broadcastable", "new_order")
 
     def __init__(self, input_broadcastable, new_order, inplace=False):
```

Comment:
Yeah, I guess you've mentioned this before or i understood this from one of your comment, but somehow forgot to make this change. I'll update in the upcoming commit